src/constants/status.asm

```diff
@@ -0,0 +1,13 @@
+;
+; states that a varaible for internal use can get
```

Comment:
Nitpick: IMO we can name this file `constants/collisions.asm`.  We'll probably have several collision-related constants to put in it later.

Reply:
sounds reasonable. Also some constants in gameplay belong in a different file. My naming for that is not the best, so I let this work do someone else.

Reply:
Allright. I renamed the file, and pushed it to this PR.